src/domains/citizen-auth-app/04_function_lollipop.tf

```diff
@@ -382,11 +366,10 @@ resource "azurerm_monitor_autoscale_setting" "function_lollipop" {
 # Alerts
 # ---------------------------------
 
-resource "azurerm_monitor_scheduled_query_rules_alert_v2" "alert_function_lollipop_HandlePubKeyRevoke_failure" {
-  count = var.lollipop_enabled ? 1 : 0
+resource "azurerm_monitor_scheduled_query_rules_alert_v2" "alert_function_lollipop_itn_HandlePubKeyRevoke_failure" {
 
-  name                = "[${upper(var.domain)}|${module.function_lollipop[0].name}] The revocation of one or more PubKeys has failed"
-  resource_group_name = azurerm_resource_group.lollipop_rg[0].name
+  name                = "[${upper(var.domain)}|${module.function_lollipop_itn.name}] The revocation of one or more PubKeys has failed"
```

Comment:
use progressive number for the naming of the resources instead of `itn` suffix

Reply:
Done in [a357e03](https://github.com/pagopa/io-infra/pull/1037/commits/a357e03aacb10507ae29d7b78542a105b6441454)